src/web/lib/components/ThemeColorsEditor/index.js

```diff
@@ -90,34 +117,75 @@ class ThemeColorsEditor extends React.Component {
                 >
                   <span
                     className="theme-unit__swatch"
-                    style={{ backgroundColor: colorToCSS(color) }}
-                    title={colorLabels[name]}
+                    style={color ? {backgroundColor: colorToCSS(color)} : {borderStyle: "dashed"}}
+                    title={labels[name]}
                   />
-                  <span className="theme-unit__label" title={colorLabels[name]}>
-                    {colorLabels[name]}
+
+                  <span className="theme-unit__label" title={labels[name]}>
+                    {labels[name]}
                   </span>
+
                 </li>
               ];
             })}
           </ul>
           <StorageSpaceInformation />
         </div>
         <div className="theme-colors-editor__picker">
-          {selectedColor && (
+          {selectedColor && advancedColors && 
+            <div className="theme-colors-editor__options">
+              <label>
+                <input type="radio" value="default" checked={!colors[selectedColor]}
+                        onChange={ev => this.handleClearColor(selectedColor)}/>
+                use Firefox&#39;s default style for this color
+              </label>
+              <label>
+                <input type="radio" value="other" checked={!!colors[selectedColor]}
+                        onChange={ev =>
+                          this.selectLastColor(selectedColor)
+                        }/>
+                or select a color:
+              </label>
+            </div>           
+          }
+          {selectedColor &&
             <SketchPicker
+              className={advancedColors && !colors[selectedColor] ? "theme-colors-editor__disabled" : ""}
               color={colors[selectedColor]}
```

Comment:
The color picker is still jumping. Use this to fix that:

`colors[selectedColor] || this.lastSelectedColor`